### PR TITLE
Astro vs X - Remix 

### PR DESCRIPTION
**Pull request by sarah11918**
This is a short section for a Remix comparison to address #244 which includes the Remix Docs site comparison.

TODO:

- [x]  confirm that Remix docs are, in fact, built using Remix (we're pretty sure. I'll go with it.)

- [x]  write a short description of Remix to match what is in the other comparisons

### CODE REVIEW

**Comment by sarah11918** [comment]
Shamelessly adding this here, but we need to transform it to more closely match our other basic descriptions, make it not-as-awesome-as-Ryan'ified.  ;)

remix

**Comment by delucis** [comment]
Thanks Sarah! Let me know if you need any pointers Astro-ifying the Lighthouse report.

File: src/pages/en/comparing-astro-vs-other-tools.md [hunk 1]
```diff
@@ -222,6 +222,18 @@ Nuxt has great built-in image optimizations. While Astro does not have a compara
...
+## Remix vs. Astro
+
+[Remix](https://remix.run) is a popular website and application framework for React.
+
+
+#### Case Study: Building a Documentation Website
+
+[remix.run/docs](https://remix.run/docs/) is the official Remix documentation website, built with Remix. The website offers a similar enough design and feature set to compare against the official Astro documentation website. This gives us a **_rough, real-world_** comparison between the two site builders for this common use-case.
+
+- **Remix performance score**: 89 out of 100 [(full audit)](/lighthouse/remix.html)
```
Comment (sarah11918) [inline src/pages/en/comparing-astro-vs-other-tools.md:234]:
```suggestion
- **Remix performance score**: 89 out of 100 [(full audit)](/lighthouse/remix)
```

Comment (sarah11918) [reply src/pages/en/comparing-astro-vs-other-tools.md:234]: update remix link for new structure

File: src/pages/en/comparing-astro-vs-other-tools.md [hunk 2]
```diff
@@ -222,6 +222,18 @@ Nuxt has great built-in image optimizations. While Astro does not have a compara
...
+
+[Remix](https://remix.run) is a popular website and application framework for React.
+
+
+#### Case Study: Building a Documentation Website
+
+[remix.run/docs](https://remix.run/docs/) is the official Remix documentation website, built with Remix. The website offers a similar enough design and feature set to compare against the official Astro documentation website. This gives us a **_rough, real-world_** comparison between the two site builders for this common use-case.
+
+- **Remix performance score**: 89 out of 100 [(full audit)](/lighthouse/remix)
+- **Astro performance score**: 92 out of 100 [(full audit)](/lighthouse/astro.html)
```
Comment (sarah11918) [inline src/pages/en/comparing-astro-vs-other-tools.md:235]:
```suggestion
- **Astro performance score**: 92 out of 100 [(full audit)](/lighthouse/astro)
```

**Comment by sarah11918** [comment]
Alright, I give up! My checks finally pass, but now the deploy fails. Clearly, there's more to do to that remix file than just renaming it and changing the scripts to `is:inline`.

The file is in the right spot, with the right extension, but I'm not sure what @delucis & @hippotastic did to these .html files to get them to be proper .astro pages. NOW I will ask for help.  :)

**Review by Chrissdroid — APPROVED** [review]
SGTM! nothing to contest or improve, super cool ! 👍

**Review by delucis — APPROVED** [review]
Well done getting this working! The text reads well 👍

**Comment by sarah11918** [comment]
Thanks @Chrissdroid @delucis . This completes the page for me! 🥳

I think I'll go with this, and let any further content additions be driven by being wrong on the internent" strategy: if someone says, "Why didn't you say that Remix does ...?" then I can add it! 😉

**Pull request merged**